adapter/redisquota/driver.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017 Google Inc.
```

Comment:
It's not clear to me that there is any value in using interfaces here instead of directly manipulating the underlying types. This would reduce the overall amount of code, would eliminate some casting, and would likely run faster too).

In other words, instead of having ConnPool and connPoolImpl, you can just have the single struct type connPool and use it in this module.